images/win/scripts/Installers/Install-SQLExpress.ps1

```diff
@@ -0,0 +1,114 @@
+################################################################################
+##  File:  Install-SQLExpress.ps1
+##  Desc:  Install SQL Express for Windows
+################################################################################
+
+function Download-FullSQLPackage {
+    param(
+        [String]$InstallerPath,
+        [String]$Arguments = ("/MEDIAPATH=$downloadPath", "/MEDIATYPE=Core","/Action=Download", "/QUIET")
```

Comment:
I think `$downloadPath` should be parameter for function.
Also why we need to pass arguments as parameters for function?